projects/ikia/seeds/seeds.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# These seeds need to be loaded first, and in order.
+%w[questionnaires protocols organizations teams].each do |seed_directory|
```

Comment:
eigenlijk moeten we dit ook in de hogere seeds file zetten, denk je niet? Het wordt nu vaak gecopy paste, maar standaard alle seeds in de seed_directory lezen lijkt me eigenlijk wel logisch. Dit is verder ook prima

Reply:
Goed ik pas het aan.

Reply:
Fixed.